package.json

```diff
@@ -149,13 +149,6 @@
       }
     },
     "preset": "ts-jest",
-    "globals": {
```

Comment:
Using same target as project (`es5`) so I can have a regression test for this issue